test/e2e/specs/fakerp/sync.go

```diff
@@ -18,12 +20,28 @@ var _ = Describe("sync pod tests [EveryPR]", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(len(pods.Items)).To(BeNumerically(">", 0))
 
-		b, err := sanity.Checker.Client.Admin.CoreV1.Pods("kube-system").GetLogs(pods.Items[0].Name, &corev1.PodLogOptions{}).DoRaw()
-		Expect(err).ToNot(HaveOccurred())
+		for {
+			b, err := sanity.Checker.Client.Admin.CoreV1.Pods("kube-system").GetLogs(pods.Items[0].Name, &corev1.PodLogOptions{}).DoRaw()
+			Expect(err).ToNot(HaveOccurred())
 
-		rx := regexp.MustCompile(`(?ms)^[^\n]* msg="starting sync"[^\n]*$.*?^[^\n]* msg="sync done"[^\n]*$`)
-		runs := rx.FindAllString(string(b), -1)
-		// check for constantly updated objects
-		Expect(runs).To(ContainElement(Not(ContainSubstring("Update"))))
+			rx := regexp.MustCompile(`(?ms)^[^\n]* msg="starting sync"[^\n]*$.*?^[^\n]* msg="sync done"[^\n]*$`)
```

Comment:
Small nit: we're compiling this regex every loop... should we pull this out?

Reply:
sure, i assumed it was e2e testing so performance was not  an issue. but i can pull it out